t/Utils.t

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Test::More;
+
+use Utils;
+
+local %ENV = %ENV;
+
+subtest "Get_Remote_User_Names" => sub {
+  my $save_remote_user = $ENV{REMOTE_USER};
+  my $save_eppn = $ENV{eppn};
+
+  subtest "with logged-in user" => sub {
+    $ENV{eppn} = 'EPPN@default.invalid';
+    $ENV{REMOTE_USER} = 'REMOTE_USER@default.invalid';
+    my @names = Utils::Get_Remote_User_Names();
+    ok(1 <= scalar grep(/remote_user/, @names), "contains lowercase REMOTE_USER");
+    ok(1 <= scalar grep(/REMOTE_USER/, @names), "contains case-preserved REMOTE_USER");
```

Comment:
Or was there already functionality to have both the case-preserved and lowercased `REMOTE_USER`?